contracts/DxMgnPool.sol

```diff
@@ -138,7 +138,7 @@ contract DxMgnPool is Ownable {
         // Don't revert if wen can't claimSellerFunds
         address(dx).call(abi.encodeWithSignature("claimSellerFunds(address,address,address,uint256)", secondaryToken, depositToken, address(this), lastParticipatedAuctionIndex));
         mgnToken.unlockTokens();
-        totalDeposit = dx.balances(address(depositToken), address(this));
+        totalDeposit = dx.balances(address(depositToken), address(this)) + depositToken.balanceOf(address(this));
```

Comment:
I believe I can attack this contract now, by "donating" some deposit token to it after the pooling has ended, but before the tokens have claimed. Then, `totalDeposit > dx.balances(address(depositToken), address(this)` and thus `dx.withdraw(address(depositToken), totalDeposit)` will revert (https://github.com/gnosis/dx-contracts/blob/master/contracts/DutchExchange.sol#L356)